core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -86,6 +86,13 @@
                 </span>
               </div>
             </div>
+            <div class="text-right">
+              <button type="button" class="btn btn-success"
```

Comment:
I'm not sure this should be a 'success' button. I think Jenna commented on that, too.